CMakeLists.txt

```diff
@@ -100,3 +102,14 @@ if (ANDROID_ABI)
   )
 endif()
 
+install(TARGETS fbjni EXPORT fbjniLibraryConfig
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}) #For windows
+
+install(EXPORT fbjniLibraryConfig DESTINATION share/cmake/fbjni
+  FILE fbjniLibraryConfig.cmake)
+
+if(MSVC)
+  install(TARGETS fbjni DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
One thing you are missing is that you forget to install the PDB fles on debug builds. See https://github.com/pytorch/FBGEMM/blob/master/CMakeLists.txt#L253-L255